### PR TITLE
make sure [registry]admin covers [registry]edit

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://github.com/openshift/origin/issues/9000

@liggitt 
[test]

### CODE REVIEW

**Comment by liggitt** [comment]
Add a check to policy_test.go that makes sure all default policy rules set an API group?

**Comment by liggitt** [comment]
extra test request, LGTM otherwise

**Comment by deads2k** [comment]
Spoke in person.  Will open a separate pull to bring our roles into sync.

[merge]

File: pkg/cmd/server/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -861,7 +869,27 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		}
 	}
 
-	return roles
+	// TODO roundtrip roles to pick up defaulting for API groups.  Without this, the covers check in reconcile-cluster-roles will fail.
+	// we can remove this again once everything gets group qualified and we have unit tests enforcing that.  other pulls are in
+	// progress to do that.
+	versionedRoles := []authorizationapiv1.ClusterRole{}
```
Comment (deads2k) [inline pkg/cmd/server/bootstrappolicy/policy.go]: @liggitt Removed the merge tag.  I had to add this.  Comment is above.  its temporary, but significant enough to deserve a second look.

**Comment by deads2k** [comment]
[merge]

File: pkg/cmd/server/bootstrappolicy/policy.go (hunk 1 above)
Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/policy.go]: as long as it's extremely temporary

**Comment by liggitt** [comment]
test failures:

* integration test router flake: #8502

* authentication/scopedtokens.test/cmd/authentication.sh:30: executing 'oc get projects --token="${listprojecttoken}"' expecting success and text 'cmd-authentication'

    ```
=== BEGIN TEST CASE ===
test/cmd/authentication.sh:30: executing 'oc get projects --token="${listprojecttoken}"' expecting success and text 'cmd-authentication'
FAILURE after 0.188s: test/cmd/authentication.sh:30: executing 'oc get projects --token="${listprojecttoken}"' expecting success and text 'cmd-authentication': the command returned the wrong error code; the output content test failed
There was no output from the command.
Standard error from the command:
Error from server: scopes [user:list-projects] prevent this action; User "scoped-user" cannot list all projects in the cluster
=== END TEST CASE ===
```

* origin/pkg/authorization/authorizer.TestAPIGroupDeny

   ```
--- FAIL: TestAPIGroupDeny (0.00s)
panic: api.IsPersonalSubjectAccessReview is not a registered type and cannot be converted into version "unknown/unknown" [recovered]
	panic: api.IsPersonalSubjectAccessReview is not a registered type and cannot be converted into version "unknown/unknown"
```

**Comment by deads2k** [comment]
[merge]

**Comment by deads2k** [comment]
conformance problem: https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_conformance/1254/

**Comment by deads2k** [comment]
conformance wedged again: https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4069/

**Comment by deads2k** [comment]
and conformance flakes again.

@liggitt I'm not crazy, that test job is green, right?

**Comment by deads2k** [comment]
this time integration hung: https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4085/

ending with
```

There was no error output from the command.
Running test/end-to-end/core.sh:176: executing 'oc delete pod cli-with-token-2' expecting success...
SUCCESS after 0.146s: test/end-to-end/core.sh:176: executing 'oc delete pod cli-with-token-2' expecting success
Standard output from the command:
pod "cli-with-token-2" deleted

There was no error output from the command.
```

**Comment by deads2k** [comment]
And now a test and problem to look at tomorrow https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/4091/

re[merge]

**Comment by deads2k** [comment]
Yum failures 

re[merge]

**Pull request merged**